src/main/asciidoc/java/index.adoc

```diff
@@ -202,6 +202,69 @@ oc policy add-role-to-user view system:serviceaccount:$(oc project -q):default -
 
 Further configuration details are available on the https://github.com/jgroups-extras/jgroups-kubernetes[Kubernetes discovery protocol for JGroups repository].
 
+=== Rolling updates
```

Comment:
You may find Infinispan Rolling Update chapter in our guide https://github.com/infinispan/infinispan/blob/master/documentation/src/main/asciidoc/user_guide/cloud.adoc#using-kubernetes-and-openshift-rolling-updates (just in case).

Reply:
Thanks for the pointer. I will a link in the document.